cli_tools_tests/e2e/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -208,6 +211,38 @@ func runImageImportOSFromImageTest(ctx context.Context, testCase *junitxml.TestC
 	runImportTest(ctx, argsMap[testType], testType, testProjectConfig.TestProjectID, imageName, logger, testCase)
 }
 
+func runImageImportOSFromVHDImageTest(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType e2e.CLITestType) {
+
+	suffix := path.RandString(5)
+	imageName := "e2e-test-image-import-os-from-vhd-image-" + suffix
+
+	argsMap := map[e2e.CLITestType][]string{
+		e2e.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-image_name=%v", imageName), "-os=ubuntu-1804",
+			fmt.Sprintf("-source_file=gs://%v-test-image/ubuntu-1804-azure.vhd", testProjectConfig.TestProjectID),
```

Comment:
At this point in your project, I'd recommend adding your test to this file (either as a new row in the existing table-driven test, or as a new function. Personally I'd do a new function to make it simpler to run locally)

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/cli_tools_tests/module/import/inflater_test.go#L35

 These are fast to run, allow you to customize the execution environment, and support using a local debugger. Directions on running these tests: https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/cli_tools_tests/module

I'm fine if you want to also keep the test in this file; I'll leave it up to you!

Reply:
Are inflater_test.go run in our test dashboard? For VHD tests, we want to run them regularly to gather some usage data for the PD team.

Reply:
Still working on automation for the module tests.

The least verbose way to add a prow test is a row in this table. I'd argue it's a better choice for this use case, since what's written here does four identical imports on every execution. For the same cost, you can do four different VHDs in the file below.

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/4f42f758697b956153b95d4a216b5154f6636246/cli_tools_tests/e2e/gce_image_import_export/test_suites/import/import_tests.go#L114

`cli_tests` is good when a feature introduces new flags, and we want to validate behavior across versions of gcloud and `gce_vm_image_import`. Custom service accounts, for example.

All of that said, we have a daisy integ test that uses VHD currently:

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/5b81051e3e719b4db620a27ec8275b1d6f48fced/daisy_integration_tests/ubuntu_1804_azure.wf.json#L11

Reply:
@dntczdx Is it possible to use the existing daisy e2e test to see the results on testgrid? Or will the daisy integration test not be a good indicator for VHD import health?

Reply:
the existing daisy e2e test directly runs daisy workflow, which is not executing CLI code. Our new VHD logic is in CLI tool. So we can't leverage that. Let's stick on e2e.